### PR TITLE
Pract.07: Add task 09 | Krasimir Ronkov

### PR DESCRIPTION
**Pull request by Krasimir-ronkov**

### CODE REVIEW

**Comment by Justsvetoslavov** [comment]
PR description is missing

File: SI/Sem.07/Pract.07/References/task09.cpp [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    int b_copy = b;
+    int count = 0;
+    while(b_copy!=0) {
+        count++;
+        b_copy /= 10;
+    }
+    int pow = 1;
+    for (int i = 0; i < count;i++) {
+        pow *= 10;
+    }
```
Comment (GeorgiTerziev02) [inline SI/Sem.07/Pract.07/References/task09.cpp:33]: You can calculate pow in the while loop above, the for is not needed

**Review by GeorgiTerziev02 — APPROVED** [review]
You can have better names for your variables, also you can create some functions so that your code can be more readable

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]
Address @GeorgiTerziev02 comment.
Remove the comments.
Move the code to appropriate functions.

**Pull request closed without merging**